pkg/server/commands/connectedobjects/connected_objects.go

```diff
@@ -341,6 +355,7 @@ func (c *ConnectedObjectsQuery) reverseExpandTupleToUserset(
 		return err
 	}
 	defer iter.Stop()
+	atomic.AddUint32(resolutionMetadata.QueryCount, 1)
```

Comment:
Shouldn't we increase _before_ we call the database? Right now, if the db call errors out, we won't coun it

Reply:
Technically, we should increase before calling the database.  However, for now, we don't return query count in case of error.  But nonetheless, I will update the code to have counter before the error out path.